src/index.ts

```diff
@@ -13,6 +13,27 @@ const S = [0n, 1n, 2n, 4n, 8n, 16n];
 type Quadbin = bigint;
 type Tile = {x: number; y: number; z: number};
 
+function cellToBoundingBox(cell: bigint) {
```

Comment:
This seems to be very similar functionality to what we have here: https://github.com/visgl/deck.gl/blob/master/modules/carto/src/layers/quadbin-utils.ts

Could you use that code here, so we can eventually use this utility in deck also and have the same logic in all places?

Reply:
Absolutely!

Reply:
It's only `worldToLngLat` which is similar to the functions you already have here. I think it is much safer to use exactly the same functions as deck is otherwise you can get numerical differences